src/components/SearchCard/SearchCard.tsx

```diff
@@ -0,0 +1,20 @@
+import styles from './SearchCard.module.css';
+import { SearchCardProps } from '../../lib/types';
+
+export default function SearchCard({ searchResults }: SearchCardProps) {
+  return (
+    <>
+      {searchResults.map((searchCard, index) => (
+        <article className={styles.searchCard} key={index}>
+          <img className={styles.albumCover} src={searchCard.cover} />
+          <div className={styles.albumInfo}>
+            <h2 className={styles.title}>{searchCard.title}</h2>
+            <p className={styles.inCollection}>
+              {searchCard.in_collection ? 'In Collection' : 'Not in Collection'}
+            </p>
+          </div>
+        </article>
+      ))}
+    </>
+  );
+}
```

Comment:
The `SearchCard` component is actually not a "SearchCard" because it gives you back an array, hence **several** cards. 
You have two options here: 
1. You could rename the component to `SearchCardList` or 
2. you could map the results in your `Search.tsx`.